unit-10/delElementArray/main.js

```diff
@@ -0,0 +1,19 @@
+
+const delElementArray = () => {
+	let array = ['A', 'C', 'A', 'A', 'B', 'D', 'B', 'D']
+	let array2 = []
```

Comment:
Khai báo là const.
Và làm không đúng yêu cầu đề bài, đề bài yêu cầu truyền params vào, ở đây lại khai báo trong hàm

Reply:
Bài này chưa làm được thầy ơi
```suggestion
	let array = ['A', 'C', 'A', 'A', 'B', 'D', 'B', 'D']
	let array2 = []
```